spec/jobs/reactions/reaction_create_job_spec.rb

```diff
@@ -0,0 +1,6 @@
+require "rails_helper"
+require "jobs/shared_examples/enqueues_job"
+
+RSpec.describe Reactions::ReactionCreateJob, type: :job do
+  include_examples "#enqueues_job", "reaction_create", [{ user_id: 786, reactable_id: 790, reactable_type: "Article", category: "readinglist" }]
```

Comment:
Keep in mind that the tests should also test the content of the job, take a look at `spec/jobs/messages/send_push_job_spec.rb`